static/js/panel.js

```diff
@@ -20,15 +20,17 @@ var Panel = new (class Panel {
     );
 
     if (this.permalinkNode) {
+      this.permalinkNode.title += " (double-click to copy)";
```

Comment:
I'd prefer the wording " (click twice to copy)" since "double-click" specifically refers to the thing where clickCount=2 and that's not what you're checking for.